flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -2034,6 +2034,52 @@ static void genStandaloneDo(lower::AbstractConverter &converter,
                 llvm::omp::Directive::OMPD_do, dsp);
 }
 
+static void genLoopClauses(
```

Comment:
Nit: Move this to the corresponding section of the file where other `genXyzClauses` definitions are located.